src/m365/spo/commands/folder/folder-get.spec.ts

```diff
@@ -191,6 +191,713 @@ describe(commands.FOLDER_GET, () => {
     assert.strictEqual(lastCall.url, 'https://contoso.sharepoint.com/sites/test1/_api/web/GetFolderByServerRelativeUrl(\'%2Fsites%2Ftest1%2FShared%20Documents\')');
   });
 
+  it('retrieves details of folder if folder url and withPermissions option is passed', async () => {
+    sinon.stub(request, 'get').callsFake(() => {
+      return Promise.resolve({
+        "ListItemAllFields": {
```

Comment:
we could move this huge response to a const at the top like `expectedFolderResult` and then use this const in both places to reduce the amount of duplicated code.
will do that when merging